apps/server/src/modules/authentication/services/ldap.service.ts

```diff
@@ -37,15 +37,9 @@ export class LdapService {
 			client.on('connect', () => {
 				client.bind(username, password, (err) => {
 					if (err) {
-						this.logger.debug(err);
-						reject(
-							new UnauthorizedException(
-								'User could not authenticate',
-								ErrorUtils.createHttpExceptionOptions(err, 'LdapService:connect')
-							)
-						);
+						throw new UserCouldNotAuthenticateLoggableException();
```

Comment:
I don't know if that should be thrown here or rather be passed into the reject method as before.